docs/source-pytorch/starter/installation.rst

```diff
@@ -56,6 +38,22 @@ You can also use `Conda Environments <https://docs.conda.io/projects/conda/en/la
 
 ----
 
+*****************************
+Apple Silicon (M1/M2/M3) Macs
+*****************************
+While ML related python packages are updated to work with Apple Silicon, you'll need to set 2 environment variables on install.
+
+.. code-block:: bash
+
+    # needed for M1/M2/M3
```

Comment:
This is not actual anymore. lightning installs cleanly on M1/M2/M3 even without the env vars.